docs/source/tooling/truffle.rst

```diff
@@ -56,21 +56,32 @@ If you install MythX for Truffle in this manner, **you will in addition need to
 
    };
 
-Running
--------
+Usage
+-----
 
 To run MythX for Truffle, run the following command in the root of your configured Truffle project::
 
   truffle run verify
 
 .. note:: The project must compile successfully in order for the plugin to run. If the project hasn't been compiled yet, MythX for Truffle will try to compile it first.
 
-By default, all contracts in the project will be analyzed. To analyze only some of the contracts, append them to the command::
+By default, all contracts in the project will be analyzed. To analyze only some of the contracts, use the following syntax::
 
-  truffle run verify MyContract MyContract2
+  truffle run verify contract.sol
```

Comment:
WDYT about making this `truffle run verify contract1.sol contract2.sol` to show that multiple contracts can be analyzed with one command?

Reply:
Great idea. Adding it in here.